src/Core/Auth/Models/Api/Response/DeviceAuthRequestResponseModel.cs

```diff
@@ -19,7 +18,7 @@ public static DeviceAuthRequestResponseModel From(DeviceAuthDetails deviceAuthDe
             Type = deviceAuthDetails.Type,
             Identifier = deviceAuthDetails.Identifier,
             CreationDate = deviceAuthDetails.CreationDate,
-            IsTrusted = deviceAuthDetails.IsTrusted()
+            IsTrusted = deviceAuthDetails.IsTrusted,
```

Comment:
Note for reviews: This is fixing an existing bug where the device trust status is incorrectly reported in the device list request.